frontend/src/routes/(authenticated)/admin/AdminProjects.svelte

```diff
@@ -113,4 +117,6 @@
       <RefineFilterMessage total={filteredProjects.length} showing={shownProjects.length} />
     {/if}
   {/if}
+
+  <p><span class="text-bold">TEMPORARY:</span> <button class="btn btn-warning" on:click={updateAllLexEntryCounts}> Update all lex entry counts </button>
```

Comment:
I'd be inclined to make this `<DevContent>` 🤷. We don't wnat anyone else clicking this button.
Perhaps the button on the project page as well, but that's not as important.

Reply:
Done in commit eb02fa36e36da70735eed1b2c11f800cb14178cb.